gnssanalysis/gn_download.py

```diff
@@ -339,6 +345,7 @@ def get_install_crx2rnx(override=False, verbose=False):
             logging.info(f"crx2rnx already present in {_sys.path[0]}")
 
 
+# TODO: Deprecate in favour of the contextmanager?
```

Comment:
We'll need to check where this is still used and replace it, but yep, I think we can definitely get rid of this after that